plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -287,6 +287,7 @@ private Stream<AddFileEntry> activeAddEntries(Stream<DeltaLakeTransactionLogEntr
     {
         Map<String, AddFileEntry> activeJsonEntries = new LinkedHashMap<>();
         HashSet<String> removedFiles = new HashSet<>();
+        HashSet<String> dependOnDeletionVector = new HashSet<>();
```

Comment:
i don't understand what this is for. can you document it?

nit: declare as `Set`